solidity/dashboard/src/css/double-icon.less

```diff
@@ -0,0 +1,13 @@
+.double-icon-container {
+  display: flex;
+  flex-direction: row-reverse;
+
+  .main-icon, .secondary-icon {
```

Comment:
Based on the Figam views, `main-icon` should have `filter: drop-shadow(2px 0px 4px rgba(0, 0, 0, 0.25));` prop.